eo-parser/src/main/antlr4/org/eolang/parser/Program.g4

```diff
@@ -61,7 +61,11 @@ abstraction
 attributes
   :
   LSQ
-  (attribute (SPACE attribute)*)?
+  (
+    (attribute (SPACE attribute)* (SPACE vararg)?)?
```

Comment:
@Graur Looks like in some cases we can have a reverse varargs, like:
```eo
...b
```
https://github.com/objectionary/eo/blob/master/eo-parser/src/test/resources/org/eolang/parser/packs/varargs-both-ways.yaml

Reply:
@volodya-lombrozo Yes, we can. But this lexema rule is only used for free attributes. And there is no rule for reverse vararg as a free attribute in EO. The reverse vararg is used to convert a vararg to an array.